src/utils.ts

```diff
@@ -10,14 +10,56 @@ import {execSync} from 'child_process';
 import * as mkdirp from 'mkdirp';
 const isWsl = require('is-wsl');
 
-export function defaults<T>(val: T | undefined, def: T): T {
+export enum LaunchErrorCodes {
+  ERR_LAUNCHER_PATH_NOT_SET = 'ERR_LAUNCHER_PATH_NOT_SET',
```

Comment:
any reason why you went with string enums? vs non-string enums?

Reply:
@samccone They are a little easier to debug — see https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

> string enums have the benefit that they "serialize" well. In other words, if you were debugging and had to read the runtime value of a numeric enum, the value is often opaque - it doesn’t convey any useful meaning on its own (though reverse mapping can often help), string enums allow you to give a meaningful and readable value when your code runs, independent of the name of the enum member itself.